data/World/Overworld.json

```diff
@@ -2303,10 +2359,10 @@
         "scene": "Death Mountain Crater",
         "hint": "DEATH_MOUNTAIN_CRATER",
         "locations": {
-            "DMC Near GC Pot 1": "is_adult",
-            "DMC Near GC Pot 2": "is_adult",
-            "DMC Near GC Pot 3": "is_adult",
-            "DMC Near GC Pot 4": "is_adult"
+            "DMC Near GC Pot 1": "is_adult or shuffle_empty_pots",
+            "DMC Near GC Pot 2": "is_adult or shuffle_empty_pots",
+            "DMC Near GC Pot 3": "is_adult or shuffle_empty_pots",
+            "DMC Near GC Pot 4": "is_adult or shuffle_empty_pots"
```

Comment:
What's the reason for writing the logic like this? Shouldn't it just be `True`?

Reply:
Discussed with @rrealmuto in DMs, the reason is that without `shuffle_empty_pots`, these pots are empty as child but have shuffled items as adult.